django/db/backends/base/schema.py

```diff
@@ -224,6 +224,12 @@ def column_sql(self, model, field, include_default=False):
             sql += self._collate_sql(collation)
         # Work out nullability
         null = field.null
+        # Add database default
+        if field.has_db_default():
+            default_sql, db_default = self.db_default_sql(field)
+            sql += ' DEFAULT ' + default_sql
+            params += db_default
+            include_default = False
         # If we were told to include a default value, do so
         include_default = include_default and not self.skip_default(field)
         if include_default:
```

Comment:
I think the following will crash if both `db_default` and `default` are provided?

Reply:
If `db_default` is set then `field.has_db_default()` will be true and `include_default` will be explicitly set to `False` (line 232), so this branch won't run in that case.